lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -1,6 +1,6 @@
 name: New P2 Identity
 description: >-
-  The routes a user can take to prove their identity to a medium
+  The routes a user so they can prove their identity to a medium
   confidence level (P2).
 
```

Comment:
Thoughts on renaming `M2B` events -> `NO_PHOTO_ID` in here too? might be nice for them to be aligned given they're essentially the same for P1 and P2

Reply:
I can do but we'd have to be careful with existing journeys and it may extend the work. May I do it as a follow-up task?

Reply:
Yep sounds good

Reply:
Have done this work in these PRs:
1. https://github.com/govuk-one-login/ipv-core-back/pull/2197
2. https://github.com/govuk-one-login/ipv-core-back/pull/2198